libs/protobuf/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=protobuf
 PKG_VERSION:=3.17.3
-PKG_RELEASE:=1
```

Comment:
Sort of. Ola was the only reason I was building it as shared. But with this version bump + changes, it can finally be built as static as it should. I can split up if you really want.

Reply:
I don't mind. Feel free to merge as-is if the failing powerpc tests are expected.